src/click/core.py

```diff
@@ -2011,15 +2011,7 @@ def _convert(value, level):
             if level == 0:
                 return self.type(value, self, ctx)
 
-            try:
-                iter_value = iter(value)
-            except TypeError:
-                raise TypeError(
-                    "Value for parameter with multiple = True or nargs > 1"
-                    " should be an iterable."
-                )
-
-            return tuple(_convert(x, level - 1) for x in iter_value)
+            return tuple(_convert(x, level - 1) for x in value)
```

Comment:
Looks like this change isn't related.

Reply:
No, I think I went wrong somewhere while fetching the files, I have to see.